src/network/test/network.qml

```diff
@@ -0,0 +1,169 @@
+import QtQuick
```

Comment:
I'd prefer if you made the wifi tester a little more usable like the bluetooth one, so its easier to regression test with.